lib_nbgl/src/nbgl_obj.c

```diff
@@ -1639,12 +1640,17 @@ void nbgl_refreshReset(void)
 
 /**
  * @brief This functions inits all internal of nbgl objects layer
+ * @note it is supposed to be called only from App
```

Comment:
Does it means we'll have to modify all the different Apps to add such call?

Reply:
This function is already called by all apps, through UX_INIT()